benchmarks/FSharpBenchmarks/AsyncBenchmarks.fs

```diff
@@ -324,6 +324,8 @@ module Helpers =
 [<CategoriesColumn>]
 type AsyncBenchmarks() =
 
+    let getTempFileName () =
+        Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString("n"))
```

Comment:
Just a thought: if you dump these in an `IDisposable`, you can actually delete the files after the tests. Or use setup/teardown, which I believe BenchmarkDotnet also supports.

Reply:
Yeah, i need to steal my disposabledirectory/file code from other repos